packages/antd/src/components/breadcrumb/index.tsx

```diff
@@ -38,9 +39,7 @@ export const Breadcrumb: React.FC<BreadcrumbProps> = ({
 
   const ActiveLink = routerType === "legacy" ? LegacyLink : Link;
 
-  if (breadcrumbs.length === 1) {
-    return null;
-  }
+  if (breadcrumbs.length >= minItems) return null;
```

Comment:
I believe it should be like this:

```suggestion
  if (breadcrumbs.length < minItems) return null;
```